### PR TITLE
react-hooks: lesson-1

### PR DESCRIPTION
**Pull request by bluestormmax**
- Uses state to handle the input value and renders to the page
- Sets an intitalName two ways, using default state and then passing a prop to the component

### CODE REVIEW

File: src/exercise/01.js [hunk 1]
```diff
@@ -1,29 +1,29 @@
 // useState: greeting
 // http://localhost:3000/isolated/exercise/01.js
 
-import * as React from 'react'
+import React, { useState } from 'react'
```
Comment (kevinfodness) [inline src/exercise/01.js:4]: Nice, this is how we're doing it on our projects typically. Makes the subsequent code much easier to read.

File: src/exercise/01.js [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
 
-function Greeting() {
-  // 💣 delete this variable declaration and replace it with a React.useState call
-  const name = ''
+function Greeting({initialName = ""}) {
```
Comment (kevinfodness) [inline src/exercise/01.js]: If this were on one of our projects, ESLint would complain about spaces within the curly braces and wanting the string value to use single quotes:
```suggestion
function Greeting({ initialName = '' }) {
```

**Review by kevinfodness — APPROVED** [review]
Looks good! 🍣

File: src/exercise/01.js (hunk 2 above)
Comment (bluestormmax) [reply src/exercise/01.js]: 👍

**Pull request merged**